public/js/notify.js

```diff
@@ -32,16 +32,26 @@ function setupNotification(callback) {
   }
 }
 
+const AUDIO = new Audio();
+// Maybe add the possibility for the notification to increase the sound's volume every n notifications?
+function playSound(fileName) {
+  AUDIO.src = fileName;
```

Comment:
what happens if the file is invalid/does not exist?

Reply:
In wouldn't play it but now (after I push the commit), it would post a console warning message following a 404 network error whilst the notifications still work as normal (minus the sound).